variables.tf

```diff
@@ -148,6 +148,19 @@ variable "sensitive_terraform_variables" {
   description = "An optional map with sensitive Terraform variables"
 }
 
+variable "slack_notification_triggers" {
+  type        = list(string)
+  default     = [
+    "run:created",
+    "run:planning",
+    "run:needs_attention",
+    "run:applying",
+    "run:completed",
+    "run:errored"
```

Comment:
Sorry to not challenge this earlier, but do we really want to have all of these as the default? As in, is that a sane default?

Reply:
Don't want to change the old behaviour basically.

Reply:
While that is true and understandable... Maybe it's not the best default. What do you think @shoekstra?

Reply:
I can see both sides… normally I would want people to opt in to features (default = disabled), but since it was the default behaviour I would stick with maintaining original functionality with the option to modify or disable.

Reply:
Check... Then I'll shut up... Still don't agree... But I'll shut up 😉